docs/introduction/installation/docker-installation.md

```diff
@@ -149,12 +174,16 @@ not a password.
 The format of the SSH URL is:
 
 ```
-ssh://[user@]host[:port][?key1=value1&key2=value2]
+ssh://[user@]host[:port][/path/to/installation/directory/api.sock][?key1=value1&key2=value2]
 ```
 
 All values in `[]` are optional. The user and port default values will come from
-your SSH config file (~/.ssh/config). Additional parameters can be passed in
-as query parameters (after the `?`). These are:
+your SSH config file (~/.ssh/config). The API socket path can be omitted if the
+socket path is the default `/opt/tyger/api.sock` or if the `TYGER_SOCKET_PATH`
+environment variable is set on the SSH host.
```

Comment:
We could also use the environment variable on the client. Thoughts?

Reply:
I think that might be getting too clever about it, since one could have a local docker install in addition to the remote. I think it is fine the way it is. Is it possible to specify `TYGER_SOCKET_PATH` as one of the key value pairs? Then what happens if they are not consistent? I am assuming it is not possible, so a non-issue.

Reply:
It is not currently possible to specify it as a key-value pair. Instead, you give the path to the socket as the path in the URL.